src/hub.js

```diff
@@ -665,7 +665,9 @@ function handleHubChannelJoined(entryManager, hubChannel, messageDispatch, data)
         });
     };
 
+    window.APP.hub = hub;
     updateUIForHub(hub, hubChannel);
+    scene.emit("hub_updated");
```

Comment:
Any reason not to just pass the hub here instead of storing the state globally? Alternatively just roll this as another thing that happens in `updateUIForHub` and skip the event alltogether.

Reply:
I primarily wanted to be able to access the hub object in the `tips` system, which of course is an a-frame system, not a react component. I'm pretty sure this is the first time we've had to access `hub` in a-frame code. All the rest of the react code passes `hub` data down through props.
If you have a better suggestion for how to get it to the a-frame system, I'd like to use it. I agree the global is not great.

Reply:
Had to leave the global in for the `tips` system, but I did pass the hub along with the event instead.